### DISCUSSION TITLE
Neovim is too small when launched with .desktop file

### DISCUSSION THREAD

**Discussion by poperigby** [neovim/neovim#13772] [Q&A]
When I launch Neovim through it's .desktop file, it too small, like this:
![Swappshot Sat Jan 16 16:06:02 2021](https://user-images.githubusercontent.com/20866468/104827655-f81a5e80-5814-11eb-9b81-b971d95e3523.png)

If it fullscreen it and un-fullscreen it, it fixes it.

Terminal: Alacritty
WM: Sway

**Comment by zeertzjq** [comment]
This should have been fixed by #17795.
